src/main/c/ssl.c

```diff
@@ -1615,6 +1615,12 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
         return 0;
     }
 
+    if (ssl_->session == NULL) {
```

Comment:
If we need a quick work around then consider wrapping this in a `#ifdef OPENSSL_IS_BORINGSSL` with a comment to the bug report?